src/event.c

```diff
@@ -79,6 +79,8 @@ static int add_rtcp_stats(struct odict *od_parent, const struct rtcp_stats *rs)
 
  out:
 	mem_deref(od);
+	mem_deref(tx);
+	mem_deref(rx);
```

Comment:
thanks for fixing this, I added it in a separate commit (67829d86c4c2d173807b532e97c09402415341c5)

Reply:
👍